### PR TITLE
Export state from dataset module

### PR DESCRIPTION
**Pull request by ritch**
Allows for managing state externally.

See [here](https://github.com/voxel51/fiftyone/pull/2214/files#diff-df67c1016008538fab6072e014adc3a196aeb28234134be32382edf99f41c043R41) for example usage.

### CODE REVIEW

File: app/packages/dataset/src/index.tsx [hunk 1]
```diff
@@ -25,14 +25,39 @@ const DatasetWrapper = () => {
   );
 };
 
+const EXAMPLE_VIEW = [
+  {
+    _cls: "fiftyone.core.stages.Limit",
+    kwargs: [["limit", 10]],
+    _uuid: "020b33dd-775a-4b4a-a865-4901e2e6ee43",
```
Comment (kaixi-wang) [inline app/packages/dataset/src/index.tsx:32]: is this is just a hardcoded view to demo changeView()? Can _cls be a list that positionally aligns with kwargs? Or how would you apply multiple stages?

Comment (kaixi-wang) [reply app/packages/dataset/src/index.tsx:32]: Oh yeah that makes sense, I missed the `[]`

File: app/packages/dataset/src/index.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+import { getEnvironment, RelayEnvironmentKey } from "@fiftyone/state";
 
 export function getEnvProps() {
-  return {environment: getEnvironment(), environmentKey: RelayEnvironmentKey}
-}
\ No newline at end of file
+  return { environment: getEnvironment(), environmentKey: RelayEnvironmentKey };
+}
+
+export * as fos from "@fiftyone/state";
```
Comment (imanjra) [inline app/packages/dataset/src/index.ts:8]: Nice!

**Review by imanjra — APPROVED** [review]
lgtm

**Pull request merged**